### PR TITLE
RANCHER-1201 Add dcb, circulation modules and edge-dcb

### PR DESCRIPTION
**Pull request by DmytrMoroz**

### CODE REVIEW

**Review by renatsafiulin — APPROVED** [review]

File: charts/edge-dcb/values.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+#     mountPath: /etc/sftp/users.conf
+
+#volumeMounts:
+#  secrets:
+#    - name: secure-store-props
+#      mountPath: /etc/edge
+#      secretName: ephemeral-properties
+
+config:
+  - name: edge-fqm-ephemeral-properties
```
Comment (OHaimanov) [inline charts/edge-dcb/values.yaml:97]: Please fix file name for config

File: charts/mod-dcb/values.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# Default values for mod-circulation.
```
Comment (OHaimanov) [inline charts/mod-dcb/values.yaml:1]: Please fix name in comment for all newely added charts

File: charts/mod-dcb/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+name: mod-dcb
+description: A Helm chart Folio mod-circulation for Kubernetes
```
Comment (OHaimanov) [inline charts/mod-dcb/Chart.yaml:3]: Please review and fix description in newely created charts

**Review by OHaimanov — CHANGES_REQUESTED** [review]

File: charts/mod-dcb/values.yaml (hunk 2 above)
Comment (DmytrMoroz) [reply charts/mod-dcb/values.yaml]: Done

File: charts/mod-dcb/Chart.yaml (hunk 3 above)
Comment (DmytrMoroz) [reply charts/mod-dcb/Chart.yaml]: Done

**Pull request merged**